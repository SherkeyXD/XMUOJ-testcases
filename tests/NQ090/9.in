16 16 17
5 19 5
8 14 12
16 14 4
13 12 14
19 8 18
13 16 17
0 0 0
