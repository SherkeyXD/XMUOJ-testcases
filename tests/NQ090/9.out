16
19
10
56
12
9
13
